code/modules/mob/abstract/new_player/sprite_accessories.dm

```diff
@@ -2952,16 +2953,109 @@ Follow by example and make good judgement based on length which list to include
 		species_allowed = list("Bishop Accessory Frame")
 
 		bishop_mask
-			name = "Face Mask"
+			name = "Bishop - Face Mask"
 			icon_state = "bishop_mask"
 			do_colouration = FALSE
 			body_parts = list(BP_HEAD)
 
 			bishop_triangular_mask
-				name = "Triangular Face Mask"
+				name = "Bishop - Triangular Face Mask"
 				icon_state = "bishop_triangular_mask"
 
-/*
+		bishop_panels
+			name = "Bishop - Full Body Panel Colors"
+			icon_state = "bishop_panels"
+			body_parts = list(BP_L_FOOT,BP_R_FOOT,BP_L_LEG,BP_R_LEG,BP_L_HAND,BP_R_HAND,BP_L_ARM,BP_R_ARM,BP_CHEST,BP_HEAD)
+
+			bishop_head
+				name = "Bishop - Head Panel Colors"
+				body_parts = list(BP_HEAD)
+
+			bishop_legs
+				name = "Bishop - Leg Panel Colors"
+				body_parts = list(BP_L_FOOT,BP_R_FOOT,BP_L_LEG,BP_R_LEG)
+
+			bishop_arms
+				name = "Bishop - Arm Panel Colors"
+				body_parts = list(BP_L_HAND,BP_R_HAND,BP_L_ARM,BP_R_ARM)
+
+//hephaestus g1
+	g1_primary
+		name = "G1 - Primary Panel Colors"
+		icon = 'icons/mob/human_races/markings_industrial.dmi'
+		icon_state = "g1_primary"
+		icon_blend_mode = ICON_MULTIPLY
+		is_painted = TRUE
+		body_parts = list(BP_L_FOOT,BP_R_FOOT,BP_L_LEG,BP_R_LEG,BP_L_HAND,BP_R_HAND,BP_L_ARM,BP_R_ARM,BP_GROIN,BP_CHEST,BP_HEAD)
+		species_allowed = list("Hephaestus G1 Industrial Frame")
+
+		/*g1_secondary
```

Comment:
remove or finish the commented out code and you should be gucci

Reply:
Removed commented code and removed the sprites associated with them in latest commit.